packages/client/src/editor/components/StickyNote.tsx

```diff
@@ -13,24 +13,9 @@ interface NoteProps {
 const StickyNote: React.FunctionComponent<NoteProps> = props => {
   const { note } = props;
 
-  const [mouseDown, setMouseDown] = useState(false);
-  const [pos, setPos] = useState({x: note.position.x, y: note.position.y});
-  const ref = useRef<HTMLDivElement>(null);
-
-  useEffect(() => {
-    if (ref.current) {
-      //ref.current.style.transform = `translate(${pos.x}px, ${pos.y}px)`; 
-      ref.current.style.setProperty('left', (note.position.x + pos.x).toString());
-      ref.current.style.setProperty('top', (note.position.y + pos.y).toString());
-    }
-    props.move(note.id, pos.x,  pos.y);
-  }, [pos])
-
-  const onMouseMove = (e: React.MouseEvent) => {
+  const onMove = (e: DraggableEvent, d: DraggableData) => {
     e.preventDefault();
```

Comment:
not sure if e.preventDefault() is needed or what it does (this was included somewhere online :D)

Reply:
Generally, it tells the browser not to do the thing it would normally do. So if you have a handler for "onclick" on a link, it tells not to actually load that link. https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault
But I don't think there is any default action for moving the mouse, so maybe it's not necessary here.
On the other hand, I think the event we are getting is something prepared by the react-draggable link and not the original of the browser, so I don't know how that changes things.